.github/workflows/ci-cd.yaml

```diff
@@ -175,13 +175,12 @@ jobs:
           mkdir -p $HOME/.config/osc
           cp /home/osc/.config/osc/oscrc $HOME/.config/osc
           /scripts/init_osc_creds.sh
-      - name: Prepare .changes file
-        # The .changes file is updated only in release creation. This current task should be improved
-        # in order to add the current rolling release notes
-        if: github.event_name == 'release'
+      - name: Prepare .changes file and prepare OBS package
         run: |
-          CHANGES_FILE=$PACKAGE_NAME.changes
-          osc checkout $OBS_PROJECT $PACKAGE_NAME $CHANGES_FILE
+          CHANGES_FILE=${{ matrix.packages }}.changes
```

Comment:
We only should update the `.changes` file on release, not always we merge to main.